roles/common/tasks/rhel-entitlements.yml

```diff
@@ -69,14 +69,14 @@
 
 - name: Disable all rhsm repos
   command: subscription-manager repos --disable '*'
-  when: replace_repos == True
+  when: replace_repos == "True"
   # This produces an absurd amount of useless output
   no_log: True
 
 - name: Enable necessary rhsm repos
   command: subscription-manager repos --enable {{ item }}
   with_items: rhsm_repos
-  when: replace_repos == True and rhsm_repos|length > 0
+  when: replace_repos == "True" and rhsm_repos|length > 0
```

Comment:
We explicitly set ``replace_repos`` to either True or False.  From my understand (true, True, False and false) will all be made a boolean by jinja2.  If we wanted we could do an explicit cast to a bool with: ``when: "{{ replace_repos|bool }}"`` I believe.

See: http://docs.ansible.com/ansible/playbooks_conditionals.html#the-when-statement